package.json

```
@@ -65,6 +65,7 @@
 		"@types/estree": "1.0.5",
 		"@types/istanbul-lib-report": "3.0.0",
 		"@types/mime": "2.0.3",
+		"@types/node": "22.10.2",
```

Gutenberg declares node >= 20, it may be better to use the 20 major version (the lowest supported version). (This will require package-lock update as well)

https://github.com/WordPress/gutenberg/blob/5c7f039bf6804449ec09723f37179d76355c14d7/package.json#L17-L20

```suggestion
		"@types/node": "^20.17.10",
```

---

I agree. Done. Thanks